eng/packaging.targets

```diff
@@ -42,4 +54,76 @@
     <MSBuild Projects="$(PkgProjPath)"
              Targets="Build" />
   </Target>
+    
+  <Target Name="AddRefAssemblies" Condition="'$(IncludeRefInPackage)' == 'true' and '$(TargetsAnyOS)' == 'true' and '@(ResolvedMatchingContract)' != ''"> 
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="@(ResolvedMatchingContract)">
+        <RefAssemblyOutput>true</RefAssemblyOutput>
+        <PackagePath>ref/$([System.String]::new('%(ResolvedMatchingContract.SetTargetFramework)').Substring(16))</PackagePath>
+        <PackagePathWithFilename>ref/$([System.String]::new('%(ResolvedMatchingContract.SetTargetFramework)').Substring(16))/%(ResolvedMatchingContract.Filename).dll</PackagePathWithFilename>
+      </TfmSpecificPackageFile>
+      <TfmSpecificPackageFile Include="$(XmlDocFileRoot)1033\$(TargetName).xml" Condition="Exists('$(XmlDocFileRoot)1033\$(TargetName).xml')">
+        <RefAssemblyOutput>true</RefAssemblyOutput>
+        <PackagePath>ref/$([System.String]::new('%(ResolvedMatchingContract.SetTargetFramework)').Substring(16))</PackagePath>
+        <PackagePathWithFilename>ref/$([System.String]::new('%(ResolvedMatchingContract.SetTargetFramework)').Substring(16))/%(ResolvedMatchingContract.Filename).xml</PackagePathWithFilename>
+      </TfmSpecificPackageFile>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="AddRuntimeSpecificAssemblies"
+          DependsOnTargets="BuiltProjectOutputGroup;SatelliteDllsProjectOutputGroup;_AddPriFileToPackBuildOutput;$(TargetsForTfmSpecificBuildOutput)"
+          Condition="'$(TargetsAnyOS)' != 'true' and '$(ExcludeFromPackage)' == ''">
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="@(SatelliteDllsProjectOutputGroupOutput);
+                            @(BuiltProjectOutputGroupOutput);
+                            @(_PathToPriFile)">
```

Comment:
What is a Pri file? I'm slightly concerned about the internal target usage.

Reply:
Projects with target framework like UWP, Win8, wpa81 produce a Pri file and is part of packages.

This target is meant to be exactly similar to https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets#L435
with the path being different.

We can remove it as we dont use it in our repo.

Reply:
Sounds fine to me to remove it as we don't produce any wpa81/win8/uap specific assets in the repo anymore.